cardano-node/src/Cardano/Node/Configuration/Socket.hs

```diff
@@ -57,7 +57,8 @@ getSocketOrSocketInfoAddr (SocketInfo info)   =
 data SocketConfigError
     = NoPublicSocketGiven
     | NoLocalSocketGiven
-    | ClashingPublicSocketGiven
+    | ClashingPublicIpv4SocketGiven
+    | ClashingPublicIpv6SocketGiven
```

Comment:
I split this into two cases because from output it is impossible to figure out which of the two errors it is.